src/Neighbor/PairwiseFieldElementAccessor.hh

```diff
@@ -0,0 +1,43 @@
+//---------------------------------Spheral++----------------------------------//
```

Comment:
I had to include the PairwiseField.hh header to SpheralPseudoScript to resolve a "error: implicit instantiation of undefined template" error; I also had to add this header to the CMakeLists.txt file so that it would be installed.

Reply:
Fixed